src/components/ProjectCreation.tsx

```
@@ -0,0 +1,814 @@
+                      <div className="font-medium text-gray-900">{framework.label}</div>
+                      <div className="text-sm text-gray-600">{framework.description}</div>
+                    </div>
+                  </label>
+                ))}
+              </div>
+            </div>
+          </div>
+        )
+
+      case 'framework':
+        const selectedFramework = FRAMEWORK_OPTIONS.find(f => f.value === formData.frameworkType)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)